letsencrypt/rootfs/etc/services.d/lets-encrypt/run

```diff
@@ -15,6 +15,7 @@ CHALLENGE=$(bashio::config 'challenge')
 DNS_PROVIDER=$(bashio::config 'dns.provider')
 ACME_SERVER=$(bashio::config 'acme_server')
 ACME_ROOT_CA=$(bashio::config 'acme_root_ca_cert')
+CHAIN=$(bashio::config 'chain')
```

Comment:
This must default to `ISRG Root X1`. Most modern devices do not trust `DST Root CA X3` anymore which means certificates generated with the old default chain are useless for a majority of users. As is this is a breaking change that will break a vast majority of users on upgrade to satisfy a few.

Either this variable has to default to `ISRG Root X1` when omitted or add `chain: ISRG Root X1` to the `options` in `config.yaml` (which causes `chain` to default to that when not explicitly provided by the user).

I'll note that you are right the option makes no sense for a custom acme server. `preferred-chain` should be omitted when that option is used. If you want to handle that feel free else it can be done in a separate PR.